crypto/info.c

```diff
@@ -7,13 +7,61 @@
  * https://www.openssl.org/source/license.html
  */
 
-#include <stddef.h>
 #include <openssl/crypto.h>
 #include "internal/dso_conf.h"
+#include "internal/thread_once.h"
+#include "internal/cryptlib.h"
 #include "e_os.h"
 
+#if defined(__arm__) || defined(__arm) || defined(__aarch64__)
+# include "arm_arch.h"
+#endif
+
+char ossl_cpu_info_str[128] = "";
```

Comment:
Travis complains, you need to qualify this with a `extern char ossl_cpu_info_str[128];` first

Reply:
Ah, thanks!